osbuild/buildroot.py

```diff
@@ -263,3 +270,14 @@ def run(self, argv, monitor, binds=None, readonly_binds=None):
         data.close()
 
         return CompletedBuild(proc, output)
+
+    @classmethod
+    def pollHelper(cls, events, pollerObject):
```

Comment:
nit: same as above, no camelCase. Also just the naming is slightly off. maybe move the call to `poll` into it too and call it `read_with_timeout`. Alternatively we could create a `Reader` class that implements `__iter__`, which internally sets up the poll object and then one could do a `for data in Reader(fd, timeout=<>)`

Reply:
Cool,
I have gone down the `read_with_timeout` route. Have a look, and if you prefer the `Reader` class route then I can switch it over :smile: